cypress/locators/advanced-color-picker/index.js

```diff
@@ -0,0 +1,10 @@
+import {
+  EXPERIMENTAL_SIMPLE_COLOR_PICKER,
+  ADVANCED_COLOR_PICKER_CELL,
+} from './locators';
+
+// component preview locators
+export const experimentalSimpleColorPickerInput = index => cy.iFrame(
```

Comment:
Easier will be set `[data-component="simple-color"]` as locator in `locator.js` file and the use it as above
```suggestion
export const experimentalSimpleColorPickerInput = index => cy.iFrame(SIMPLE_COLOR).find('input').eq(index);
```

Reply:
applied slightly different implementation, thanks to Ihor. 👍